src/components/Icon/Icon.jsx

```diff
@@ -47,7 +48,7 @@ const Icon = forwardRef(
         <IconComponent
           size={iconSize.toString()}
           onClick={onClick}
-          tabIndex={tabindex}
+          tabIndex={externalTabIndex || tabindex}
```

Comment:
this could be an issue if the external is 0 and the tabIndex might be for example -1 ?